lxd/api.go

```diff
@@ -149,3 +151,109 @@ func queryParam(request *http.Request, key string) string {
 
 	return values.Get(key)
 }
+
+type FilterEntry struct {
```

Comment:
Can you move the changes from this file into a `api_filter.go` file?

Reply:
As I'm suggesting moving some of the logic below to the `lxd/instance` package and that package will need access to the functions below, you'll actually need to move those functions to a separate Go package, say `lxd/filter`.

That way `lxd/instance` can import `lxd/filter` to use those functions and the main package can call the instance filter functions from `lxd/instance`.